charforge-gui/backend/app/api/training.py

```
@@ -190,7 +190,7 @@ async def start_training(
     request: TrainingRequest,
     background_tasks: BackgroundTasks,
     db: Session = Depends(get_db),
-    current_user: User = Depends(get_current_active_user)
+    current_user: User = Depends(get_current_user_optional)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)